spec.bs

```diff
@@ -1219,6 +1228,22 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
 
 </div>
 
+<div algorithm>
+
+  To <dfn>validate a url macro</dfn> given a [=string=] |macro|:
+  1. If |macro| [=string/starts with=] "'${'":
+    1. If |macro| [=string/ends with=] "'}'":

```

Comment:
this can be simplified as:
`If ... then return true, otherwise return false`

Reply:
Done.